src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -1,53 +0,0 @@
-package seedu.address.model.person;
-
-import static java.util.Objects.requireNonNull;
-import static seedu.address.commons.util.AppUtil.checkArgument;
-
-/**
- * Represents a Person's phone number in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
- */
-public class Phone {
```

Comment:
I see the Expense Tracker team has decided to remove the Phone object from the Person object. If so, do remember to update the model diagram, as well as the UserGuide, as these still contain the Phone as a relevant class attribute.

Reply:
Thanks for the reminder, we will update the guide and diagram.

Reply:
Got it. Will do. Thanks for reminding.